routing/routes.go

```diff
@@ -219,6 +219,21 @@ func NewRoutes() (r *Routes) {
 			handlers.ServiceMetrics,
 			true,
 		},
+		{
+			// Supported query parameters:
```

Comment:
Note I've created a task to swaggerize all that : https://issues.jboss.org/browse/KIALI-1329